### DISCUSSION TITLE
You are marked as spam, and therefore cannot authorize a third party application.?

### DISCUSSION THREAD

**Discussion by Sazy67** [community/community#114534] [New to GitHub]
I don't understand what happened to my account. Can you tell me what to do now? I didn't want to make a transaction that would be spam. I'm new, I'm trying some

**Comment by benogoulart** [comment]
Hi! This message usually means that GitHub's automated anti-spam system has temporarily flagged your account. It doesn't necessarily mean you intentionally did anything wrong.

The only way to resolve it is to contact GitHub Support and ask them to review your account. You can do that here:

https://support.github.com/contact

Explain that you're a new user, that you haven't engaged in any spam activity, and that you're unable to authorize third-party applications because your account has been marked as spam.

Once GitHub reviews your account, they'll let you know if the restriction can be removed.
